themes/ddbasic/templates/opening-hours/opening_hours_week.tpl.php

```diff
@@ -8,7 +8,7 @@
   print $preface;
 endif;
 ?>
-<div class="opening-hours-week placeholder" data-nid="<?php print $node->nid; ?>">
+<div class="opening-hours-week placeholder" data-nid="<?php print $node->nid; ?>" data-tid="<?php print $tid; ?>">
```

Comment:
Why is this change relevant to solving the issue? I do not see anywhere else that `data-tid` is referenced.